test/aspect.spec.js

```diff
@@ -16,8 +16,8 @@ describe("aspect", () => {
     expect(warnSpy).toHaveBeenCalledTimes(classes.length);
     expect(warnSpy.calls.flat()).toMatchInlineSnapshot(`
       [
-        "[REPLACED] aspect-w-2 -> use fractions instead, e.g. aspect-4/3",
-        "[REPLACED] aspect-h-4 -> use fractions instead, e.g. aspect-4/3",
+        "\x1b[93m[REPLACED]\x1b[0m aspect-w-2 -> use fractions instead, e.g. aspect-4/3",
+        "\x1b[93m[REPLACED]\x1b[0m aspect-h-4 -> use fractions instead, e.g. aspect-4/3",
```

Comment:
`\x1b[` -part was not on the line for example https://github.com/warp-ds/preset-migrate/pull/11/files#diff-4a3ef178d4db5cec9d054131ad2656a9385f8756a11ec5c29d6e66b1504c7807R27

Reply:
Here: `\x1b[93m[REPLACED]\x1b[0m`
Some other places: `[93m[REPLACED][0m`

Reply:
Apparently Vitest that encodes these somehow when it runs the tests... everything works fine though, but looks ugly. Not sure what to do about it. 🤷 😢